mlfinlab/structural_breaks/chow.py

```diff
@@ -3,51 +3,77 @@
 """
```

Comment:
HI @MislavSag. 

Please add the structural breaks code to a separate PR. This one should just be focused on trend scanning. Also build not passing.

Best regards
Jacques

Reply:
@Jackal08 I made new PR where I change only trend scanning. Please delete this one and check new one. I will make corrections if necessary in the new one.